src/backend/marsha/bbb/api.py

```diff
@@ -65,7 +65,9 @@ def get_permissions(self):
             ]
         elif self.action in ["retrieve"]:
             permission_classes = [
-                ResourceIsAuthenticated | IsClassroomOrganizationAdmin
+                ResourceIsAuthenticated
+                | IsClassroomOrganizationAdmin
+                | IsClassroomPlaylistAdmin
```

Comment:
I don't know if the order matter or not (how conditions are evaluated). Depending how this is implemented in DRF, checking the playlist admin should be done before Organization admin IMO.